collaborators/cj-tabaldo.html

```diff
@@ -1,34 +1,31 @@
-<!DOCTYPE html>
-<html lang="en">
+<html>
```

Comment:
Why'd you remove the `!DOCTYPE`? 

This is HTML5 basic stuff.

If you didn't know, no problem, but if you did, please explain your reasoning.

Reply:
honestly, I was not aware that I did that. I can definitely add it back in my document.